### ISSUE TITLE
requiresConsent() not working as expected

### ISSUE THREAD

**Issue by ugrupp** [balazscsaba2006/craft3-cookiebot#3]
After first install, when no cookie has been set yet, `{{ craft.cookiebot.dialogScript(craft.app.locale.id)|raw }}` does not render anything for me.
I tracked it down to `requiresConsent()` in `CookebotService.php` always returning `false`.

I looked into the function and i'm not sure, but shouldn't the logic rather be this?

```php
public function requiresConsent(): bool
{
    return !$this->isCookieSet() || ($this->isCookieSet() && '-1' !== $_COOKIE[self::COOKIE_NAME]);
}
```

Thanks for looking into this!

**Comment by balazscsaba2006** [comment]
Hi @ugrupp . I would be happy to take a look at a PR, if you have time for it.

**Comment by balazscsaba2006** [comment]
@ugrupp Closed in #5
